test/input.txt

```diff
@@ -50,10 +50,18 @@
   list
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.